### PR TITLE
Add title to video tag

### PR DESCRIPTION
**Pull request by karimJWP**
Adds the item's title to the video tag.

This allows the video's title to appear on iOS device's lock-screens.

Suggesting potential edge case scenarios would be greatly appreciated. Explored playlist with ads where some items don't have titles. 

Enables title to be displayed in the iOS lock screen, instead of video's URL

JW7-3410

### CODE REVIEW

File: src/js/view/view.js [hunk 1]
```diff
@@ -409,6 +411,12 @@ define([
             this.checkResized();
         };
 
+        function itemReady() {
+            var title = _model.get('playlistItem').title || '';
```
Comment (johnBartos) [inline src/js/view/view.js]: `const`

File: src/js/view/view.js [hunk 2]
```diff
@@ -409,6 +411,12 @@ define([
...
+            var videotag = _videoLayer.querySelector('video, audio');
```
Comment (johnBartos) [inline src/js/view/view.js]: What happens if the player is not viewable?

Comment (karimJWP) [inline src/js/view/view.js]: @johnBartos title is still set. Checked when player does not appear on page and when item is loaded when player is in background tab

**Comment by karimJWP** [comment]
@johnBartos all comments addressed

**Pull request merged**